src/bindings/lua/flux-lua.c

```diff
@@ -265,10 +265,7 @@ static int send_json_request (flux_t *h, uint32_t nodeid, uint32_t matchtag,
     if (flux_msg_set_matchtag (msg, matchtag) < 0)
         goto done;
     if (nodeid == FLUX_NODEID_UPSTREAM) {
-        uint8_t flags;
```

Comment:
commit message typo: `that the previous functions` :arrow_right: `than the previous functions`